lib/si-pkg/src/spec.rs

```diff
@@ -86,6 +88,14 @@ impl PkgSpec {
             .iter()
             .find(|func_spec| func_spec.name.as_str() == name)
     }
+
+    pub fn anonymize(&mut self) {
```

Comment:
Probably the right place for a comment saying what this does (and that you shouldn't actually use an anonymized spec for anything besides output or diff)